app/views/offers/_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= simple_form_for [@offer] do |f| %>
+  <%= f.input :title %>
+  <%= f.input :description %>
+  <%= f.input :price, as: :integer %>
+  <%= f.input :number_of_people, as: :integer %>
+  <%= f.input :category %>
```

Comment:
```suggestion
  <%= f.input :category, :collection => %w[German Italian Vietnamese Vegan French Indian Fusion American Japanese] %>
```

Reply:
would be better this way
f.input :category, collection: Offer::CATEGORY

This way categories are defined in a single place. We can add more options there